app/src/main/java/org/oppia/app/topic/ToolbarTitleListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.topic
+
+/** Interface to listen for toolbar title as they are fetched in the fragments and not activities */
+interface ToolbarTitleListener {
```

Comment:
Actually this is not the correct approach, the activity/fragment should be responsible for its own content and nothing else. So the title should be fetched by `TopicActivity` or `TopicFragment` only and not by any child fragment.

Reply:
@rt4914 I thaught that this listener would be helpful with other fragments in the future as well. Hence I wrote such a comment.
In the implementation, the responsibility of fetching the title is given to the fragment only.
Please suggest what the comment should  be if it needs to be changed

Reply:
There are two approaches to this: 
1. Transfer toolbar to topic-fragment and set data to that toolbar from its `presenter`

2. Or, currently the fragmentpresenter is fetching the information for title, you can shift that code to `activity-presenter`.

Both are correct approach.

Reply:
For the second approach,  how do I retrieve the topic title in the activity presenter? I will have to copy most of the code present within the fragment presenter for that.

Reply:
Yes, you will need to do that, meaning copy code of fragment presenter to activity presenter.